io.github.junalmeida.webapps_manager.metainfo.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>io.github.junalmeida.webapps_manager</id>
+  <icon type="stock">io.github.junalmeida.webapps_manager</icon>
+  <launchable type="desktop-id">io.github.junalmeida.webapps_manager</launchable>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <content_rating type="oars-1.1"></content_rating>
+
+  <url type="homepage">https://github.com/junalmeida/webapps_manager</url>
+  <categories>
+    <category>Network</category>
+  </categories>
+  <releases>
+    <release version="0.0.1" date="2024-12-16">
+      <url type="details">https://github.com/junalmeida/webapps_manager/releases/tag/v0.0.1</url>
+      <description>
+        <ul>
+          <li>Webapps manager QT Ui and Flatpak distribution</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+
+  <provides>
+    <binary>webapps_manager</binary>
+  </provides>
+  <developer id="io.github.junalmeida">
+    <name>Marcos Junior</name>
+  </developer>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>
+        https://raw.githubusercontent.com/junalmeida/webapps_manager/refs/heads/master/docs/screenshot.png</image>
```

Comment:
screenshot should NEVER be referenced from a tip of a branch but always from a commit or a tag.

Reply:
fixed on upstream file